setup.cfg

```diff
@@ -3,6 +3,60 @@
 [bdist_rpm]
 doc-files = doc
 
+[options]
+python_requires = >=3.6
+install_requires =
+	argparse
```

Comment:
argparse is only used once, in [`nilearn/plotting/rm_file.py`](https://github.com/nilearn/nilearn/blob/6aaf8f94433dfaf2967d69c64c0f688d7e1c812d/nilearn/plotting/rm_file.py). Could we use some other method for removing files and drop this dependency?

Reply:
not sure how it ended up in the setup but argparse is part of the python standard library